src/main/java/com/connexta/store/service/impl/StoreServiceImpl.java

```diff
@@ -40,22 +44,8 @@
   @NotNull private final StorageAdaptor fileStorageAdaptor;
   @NotNull private final StorageAdaptor irmStorageAdaptor;
   @NotNull private final IndexDatasetClient indexDatasetClient;
-
-  @Override
-  @NotNull
-  public URI createDataset(
-      @NotNull @Min(1L) @Max(10737418240L) final Long fileSize,
-      @NotBlank final String mediaType,
-      @NotBlank final String fileName,
-      @NotNull final InputStream fileInputStream)
-      throws StoreException {
-    final String datasetId = UUID.randomUUID().toString().replace("-", "");
-    fileStorageAdaptor.store(
-        fileSize, mediaType, fileInputStream, datasetId, Map.of(FILE_NAME_METADATA_KEY, fileName));
-    return UriComponentsBuilder.fromUri(retrieveUri)
-        .path(StoreController.RETRIEVE_FILE_URL_TEMPLATE)
-        .build(datasetId);
-  }
+  @NotNull private final S3StorageAdaptor metacardStorageAdaptor;
```

Comment:
`StorageAdaptor` instead of `S3StorageAdaptor`

Reply:
that was only to gain access to the retrieveStream convenience method on the impl that is now removed :+1: